tests/test_package_generation.py

```diff
@@ -36,26 +36,19 @@ def get_all_files_folders(root_path: pathlib.Path) -> set[pathlib.Path]:
 
 
 def test_package_generation(
-    tmp_path: pathlib.Path,
+    default_config_with: typing.Callable,
     generate_package: typing.Callable,
 ) -> None:
     """Test package generation."""
-    test_config = {
-        "github_owner": "test-user",
-        "project_short_description": "description",
-        "project_name": "Cookiecutter Test",
-        # Not having a git repo makes it easier to check in/out reference
-        # data files to the main python-tooling git repository
-        "initialise_git_repository": False,
-    }
-    generate_package(config=test_config, path=tmp_path)
+    # Not having a git repo makes it easier to check in/out reference
+    # data files to the main python-tooling git repository
+    config = default_config_with("initialise_git_repository", "False")
```

Comment:
```suggestion
    config = default_config_with(initialise_git_repository="False")
```
Required if we use suggestion in https://github.com/UCL-ARC/python-tooling/pull/566/files#r2224973492